packages/dts-generator/src/js-utils/ui5-metadata.js

```diff
@@ -1,16 +1,20 @@
-const { resolve } = require("path");
-const { readFile } = require("fs").promises;
-const { zipObject, map, union, flatMap } = require("lodash");
-const { emptyDir } = require("fs-extra");
+import { resolve } from "path";
+import { promises as fs } from "fs";
+import path from "path";
+import _ from "lodash";
```

Comment:
see above

Reply:
Of course. It was a proposal from an error message when I converted to ES modules step by step. I just took it over without thinking a bit more.

Reply:
Wait.. not possible because lodash is a commonjs module, see below